apis/opts/config/ulimit.go

```diff
@@ -35,6 +36,8 @@ func (u *Ulimit) String() string {
 		str = append(str, ul.String())
 	}
 
+	// make sure string list is sorted
+	sort.Strings(str)
```

Comment:
Agree, ulimit do not need slice for sort.

Reply:
@fuweid @Ace-Tang test cases need to check return string, which includes a map struct,  it means the return string is not alway the same if we don't sort the map struct firstly.

Reply:
@fuweid @Ace-Tang if this sorting is harmless, I trend to change it in here not in the case. otherwise, I can deal with it in case.

Reply:
@chuanchang , if test need sort, please sort in test, do not change code for the sort things.

Reply:
Actually it will affect the code logic. And I am afraid @Ace-Tang mentioned is correct, we can add this code in test. Changing the raw code is unnecessary.

Reply:
I have removed String() related testing and sort codes according to zhijin's advice.